models/ResumeModel.ts

```diff
@@ -13,7 +13,7 @@ export class ResumeModel extends BaseEntity {
   user: UserModel;
 
   @Column('boolean', { default: true, nullable: false })
-  isResumeVisible: boolean;
+  isResumeVisible: boolean = true;
 
```

Comment:
I think this default should be false, since the client should actually specify if it wants to share the resume publicly before doing so. That way, users wouldn't have their resumes publicly visible unless they specify (which they'd do during the upload of their resume, so it's not actually too much work on their end even if we set the default to false)